app/routes/groups_.$groupId.tsx

```diff
@@ -5,36 +5,37 @@ import { json } from '@remix-run/node';
 import { LinkButton } from '~/components/ui/forms';
 
 import { Image } from '~/components/ui/images';
+import { eventsDataPatcher } from '~/modules/events/event';
 
 export const loader: LoaderFunction = async ({ params }) => {
```

Comment:
If we use `loaderFunctionArgs` for the param types instead of `LoaderFunction` for the function, we maybe able to get better type safety.